web/client/utils/MapInfoUtils.js

```diff
@@ -21,21 +21,95 @@ import vector from './mapinfo/vector';
 import threeDTiles from './mapinfo/threeDTiles';
 
 let MapInfoUtils;
+/**
+ * Map of info modes which are used to display feature info data (identify tools).
+ * To be distinguished with INFO_FORMATS which is the map of mime types used in client server communication.
+ * These are strictly a representation of the various ways that info data is visualized.
+ * Has an N <=> N relationship with INFO_FORMATS.
+ */
+const INFO_VIEW_MODES = {
+    TEXT: "TEXT",
+    PROPERTIES: "PROPERTIES",
+    HTML: "HTML",
+    TEMPLATE: "TEMPLATE"
+};
+
+/**
+ * @returns {object} Map of views which are used to display feature info data (identify tools).
+ */
+export const getInfoViewModes = () => {
+    return {...INFO_VIEW_MODES};
+};
+/**
+ * @param {string} infoFormat the info format mime type.
```

Comment:
I should rename it `getDefaultInfoViewMode`. Please fix my example when 

```suggestion
 * Given an `infoFormat` mime-type passed, it returns the default view mode (e.g. `PROPERTIES`, `HTML`, `TEXT`) for the format selected.
 * @param {string} infoFormat the info format mime type.
```